packages/dds/tree/src/test/simple-tree/utils.ts

```diff
@@ -145,6 +147,16 @@ export function hydrate<TSchema extends ImplicitFieldSchema>(
 	return getTreeNodeForField(field) as TreeFieldFromImplicitField<TSchema>;
 }
 
+/**
+ * {@link hydrate} but unsafe initialTree
```

Comment:
```suggestion
 * {@link hydrate} but unsafe initialTree.
 * This may be required when the schema is not entirely statically typed, for example when looping over multiple test cases and thus using a imprecise schema type.
 * In such cases the "safe" version of hydrate may require `never` for the initial tree.
```